### PR TITLE
Exclude Procfile & .pre-commit-config.yaml from pii_secret_file_content_ner pre-commit check

### PR DESCRIPTION
**Pull request by WillGibson**
Because...

```console
PII Named Entity Recognition.............................................Failed
- hook id: pii_secret_file_content_ner
- exit code: 1

Using spaCY NER (https://spacy.io/) for PII checks
NER excludes will be appended to 'ner_output_file.txt'
Number of files for processing: 2
Line 21. please check 'NER' - ORG - Companies, agencies, institutions, etc.
.pre-commit-config.yaml
Line 1. please check 'src &&' - ORG - Companies, agencies, institutions, etc.
Line 2. please check 'src && celery -A config.celery' - ORG - Companies, agencies, institutions, etc.
Line 2. please check 'INFO' - ORG - Companies, agencies, institutions, etc.
Line 3. please check 'src && celery -A' - ORG - Companies, agencies, institutions, etc.
Line 3. please check 'INFO' - ORG - Companies, agencies, institutions, etc.
Procfile
NER content check failed
```

### CODE REVIEW

File: pii-secret-exclude.txt [hunk 1]
```diff
@@ -1,23 +1,25 @@
-requirements.txt
```
Comment (WillGibson) [inline pii-secret-exclude.txt:1]: I took the liberty of letting my IDE sort these so it's easier to navigate.

File: pii-secret-exclude.txt [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+.env.ci
+.env.example
+.pre-commit-config.yaml
+Procfile
```
Comment (WillGibson) [inline pii-secret-exclude.txt:4]: The two lines above are the new ones.

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/uktrade/pii-secret-check-hooks
-    rev: 0.0.0.34
+    rev: 0.0.0.35
```
Comment (WillGibson) [inline .pre-commit-config.yaml:3]: It told me to update, so I did.

**Review by CamLamb — APPROVED** [review]
LGTM

**Pull request merged**